### PR TITLE
Comparisons Documentation

### PR DESCRIPTION
**Pull request by cstml**

### CODE REVIEW

File: docs/comparison-matrix.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+| Add new builtin types                                  | 🔴                | 🟢           | 🔴              | 🟢                 | 🔴       | 🔴           |
+| Add new type semantics (e.g. different encodings)      | 🟢                | 🟢           | 🔴              | 🟢                 | 🔴       | 🟢           |
+| Manage type semantics (at language level)              | 🔴                | 🔴           | 🔴              | 🟢                 | 🔴       | 🔴           |
+| Codegen support                                        | 🟢 (Excellent)    | 🟢 (Average) | 🟢 (Excellent)  | 🟡                 | 🟢 (Bad) | 🟢 (Average) |
+| DevOps tooling - build system integration              | 🟢                | 🔴           | ❔              | 🟡                 | 🔴       | 🔴           |
+| Documentation tooling                                  | 🟢                | ❔           | 🟢              | 🔵                 | 🔴       | ❔           |
+| Formatting, linting, and development environment tools | 🟢                | 🔴           | 🟢              | 🔵                 | 🔴       | 🔴           |
```
Comment (bladyjoker) [inline docs/comparison-matrix.md:27]: Formatting is implemented :D

File: docs/comparison-matrix.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| Documentation tooling                                  | 🟢                | 🔴           | 🟢              | 🔵                 | 🔴       | ❔           |
+| Language checker API                                   | 🟢                | 🔴           | 🟢              | 🟢                 | 🔴       | 🔴           |
+| Codegen API                                            | 🟢                | 🟢           | 🔴              | 🟢                 | 🔴       | 🔴           |
+| Language specification                                 | 🟢                | 🟢           | 🟢              | 🟢                 | 🟢       | 🟢           |
+| Backwards compatibility strategy                       | 🟢                | 🔴           | 🔴              | 🔴                 | 🔴       | 🔴           |
+
```
Comment (bladyjoker) [inline docs/comparison-matrix.md:33]: Perhaps adding a chapter that elaborates on what the features are would be nice.

**Review by bladyjoker — APPROVED** [review]

File: docs/comparison-matrix.md (hunk 1 above)
Comment (cstml) [inline docs/comparison-matrix.md:27]:
```suggestion
| Formatting, linting, and development environment tools | 🟢                | 🔴           | 🟢              | 🟢                 | 🔴       | 🔴           |
```

File: docs/comparison-matrix.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+| Documentation tooling                                  | 🟢                | ❔           | 🟢              | 🔵                 | 🔴       | ❔           |
+| Formatting, linting, and development environment tools | 🟢                | 🔴           | 🟢              | 🟢                 | 🔴       | 🔴           |
+| Documentation tooling                                  | 🟢                | 🔴           | 🟢              | 🔵                 | 🔴       | ❔           |
+| Language checker API                                   | 🟢                | 🔴           | 🟢              | 🟢                 | 🔴       | 🔴           |
+| Codegen API                                            | 🟢                | 🟢           | 🔴              | 🟢                 | 🔴       | 🔴           |
+| Language specification                                 | 🟢                | 🟢           | 🟢              | 🟢                 | 🟢       | 🟢           |
+| Backwards compatibility strategy                       | 🟢                | 🔴           | 🔴              | 🔴                 | 🔴       | 🔴           |
+
```
Comment (cstml) [inline docs/comparison-matrix.md:33]:
```suggestion
:todo: add chapter elaborating on each feature
```

**Pull request merged**